### ISSUE TITLE
how to improve the training miou in ERFNet-CULane-PyTorch? 

### ISSUE THREAD

**Issue by yoga-0125** [cardwing/Codes-for-Lane-Detection#281]
Hi, 

The training miou is always around 0.44 for TuSimple dataset. There is no any improvement after adjusting the learning rate, learning rate decay, momentum, weight decay... 

Any advice? Thanks a lot.

**Comment by cardwing** [comment]
Have you tested the performance of the trained model on TuSimple? It seems that the model does not learn well from the training data. Maybe you can take the CULane trained model as initialization for TuSimple training. Other engineering tricks can be found in previous issues.

**Comment by yoga-0125** [comment]
> Have you tested the performance of the trained model on TuSimple? It seems that the model does not learn well from the training data. Maybe you can take the CULane trained model as initialization for TuSimple training. Other engineering tricks can be found in previous issues.

I tried to use your trained file as pretrained model, but TuSimple has 6 lanes labelling instead of 4 in CULane.

**Comment by cardwing** [comment]
Just load the weights of previous layers other than the last layer.

**Comment by yoga-0125** [comment]
> Just load the weights of previous layers other than the last layer.

Hello, thanks for the suggestion. I loaded and fine-tune the trained file without lane_exist & output_conv parts. However, the miou result is still around 0.44. 

The output_exist may have problems: [0,1,1,1,1,0] --> [1e-28, 1, 1, 1, 1, 1e-28] looks good
                                                             [0,0,1,1,1,1] --> [1e-28, 1, 1, 1, 1, 1e-28] position is not correct 
                                                             [0,0,1,1,1,0] --> [1e-28, 1, 1, 1, 1, 1e-28] lane # is not correct 
                                                             [0,1,1,1,0,0] --> [1e-28, 1, 1, 1, 1, 1e-28] lane # not correct

**Comment by yoga-0125** [comment]
The loss of lane_exist seems does not learn in training

**Comment by yoga-0125** [comment]
The loss of lane_exist seems does not learn significantly in training.

**Comment by cardwing** [comment]
Maybe you can slightly increase the coefficient of the lane_exist loss and train more epochs.

**Comment by yoga-0125** [comment]
Hello, 
I tried the ideas you mentioned before, unfortunately they do not work for this issue.

**Comment by foxkw** [comment]
> 
> 
> Hi,
> 
> The training miou is always around 0.44 for TuSimple dataset. There is no any improvement after adjusting the learning rate, learning rate decay, momentum, weight decay...
> 
> Any advice? Thanks a lot.

Hi, I run this project recently. why it just run 1 epoch whatever value of `b`  I set in .sh  , and  miou stayed 0.044 ,  Is the instruction of. Sh file incomplete
